load/paths/load.js

```diff
@@ -0,0 +1,40 @@
+import http from 'k6/http';
+import {check} from 'k6';
+
+export let options = {
+    stages: [
+        {duration: '1m', target: 30}
+    ],
```

Comment:
```suggestion
    stages: [
        {duration: '20s', target: 30},    
        {duration: '1m', target: 30}
        {duration: '20s', target: 0}        
    ],
```

이렇게 구성하면 1분간 VUser를 30까지 올리지 않나요
위와 같이 작성해야 하지 않을까요